app/models/course.rb

```diff
@@ -37,4 +37,16 @@ def university_based?
   def further_education?
     level == 'further_education' && subjects.any? { |s| s.subject_name == 'Further education' || s.subject_code = '41' }
   end
+
+  def travel_to_work_areas
+    travel_to_work_areas = sites.map { |site| site.london_borough || site.travel_to_work_area }.uniq
```

Comment:
so here we would do site_statuses.map(&site).map { |site| site.london_borough || site.travel_to_work_area }.uniq